gensim/test/test_tfidfmodel.py

```diff
@@ -66,6 +89,143 @@ def testPersistenceCompressed(self):
         self.assertTrue(model.idfs == model2.idfs)
         tstvec = []
         self.assertTrue(np.allclose(model[tstvec], model2[tstvec]))  # try projecting an empty vector
+
+        # Test persistence between old and new compressed model.
+        model3 = tfidfmodel.TfidfModel(self.corpus, smartirs="ntc")
+        model4 = tfidfmodel.TfidfModel.load(datapath('tfidf_model.tst.bz2'))
```

Comment:
It will be better if you add gensim version in model name ("old" in comment isn't really informative)